### PR TITLE
Prevent focus node from changing when recording finishes

### PR DESCRIPTION
**Pull request by GPayne**

### CODE REVIEW

File: src/interaction/sessionrecording.cpp [hunk 1]
```diff
@@ -633,23 +633,29 @@ void SessionRecording::cleanUpPlayback() {
...
-                    _keyframesCamera[p].focusNode);
-                if (n) {
-                    global::navigationHandler->orbitalNavigator().setFocusNode(
-                        n->identifier());
-                }
+        unsigned int p =
+            _timeline[_idxTimeline_cameraPtrPrev].idxIntoKeyframeTypeArray;
+        if (_keyframesCamera.size() > 0) {
+            const SceneGraphNode* n = scene->sceneGraphNode(
+                _keyframesCamera[p].focusNode);
```
Comment (alexanderbock) [inline src/interaction/sessionrecording.cpp:640]: The `);` should be on a new line

File: src/interaction/sessionrecording.cpp [hunk 2]
```diff
@@ -633,23 +633,29 @@ void SessionRecording::cleanUpPlayback() {
...
+            if (n) {
+                global::navigationHandler->orbitalNavigator().setFocusNode(
+                    n->identifier());
```
Comment (alexanderbock) [inline src/interaction/sessionrecording.cpp:643]: Same here

**Review by alexanderbock — APPROVED** [review]
Yey :)

**Review by WeirdRubberDuck — APPROVED** [review]
🙌

**Pull request merged**